src/draw/lv_image_decoder.c

```diff
@@ -62,6 +75,14 @@ void _lv_image_decoder_init(void)
         .create_cb = NULL,
         .free_cb = (lv_cache_free_cb_t)image_decoder_cache_free_cb,
     });
+
+    // FIXME: max_count should be configurable
```

Comment:
why not a config?
If a new config is added, image header cache code should move out of `#if LV_CACHE_DEF_SIZE > 0` ?

Reply:
It is necessary to add the independent control macro of `header cache`, but there is no time to add it. I'll add it later. 😁

Reply:
Done!